releases/2022-11-25-releasenotes.md

```diff
@@ -0,0 +1,154 @@
+content-type: application/json
+content-length: 288
+x-amzn-requestid: XXXXXX
+access-control-allow-origin: *
+x-amz-apigw-id: XXXXXX
+x-amzn-trace-id: XXXXXX
+
+{"data": null, "errors": [{"message": "Cannot query field 'listVedia_BlockItems' on type 'Query'. Did you mean 'listVendia_BlockItems', 'listVendia_FileItems', 'listVendia_FolderItems', 'listVendia_ApiKeyItems', or 'listVendia_ContractItems'?", "locations": [{"line": 1, "column": 22}]}]}%
+```
+
+2. A request with meaningless query body:
+
+```
+curl 'https://<some-domain>/graphql/' \
+    -H 'Authorization: <api-token>' \
+    -H 'content-type: application/json' \
+    -H 'accept: application/json' \
+    -i \
+    --data-raw '{"query":"query "}'
+```
+
+Again, we see a `200` status code. The message informs us that the syntax is not right which makes sense since our query body has nothing.
+
+Sample Response:
+```
+HTTP/2 200
+date: Tue, 29 Nov 2022 20:31:52 GMT
+content-type: application/json
+content-length: 124
+x-amzn-requestid: 6097318d-f50d-4097-b415-82e6a8f07652
+access-control-allow-origin: *
+x-amz-apigw-id: cYXYzEpbyK4FsBg=
+x-amzn-trace-id: Root=1-63866c38-38b964043a2d77c240da4981;Sampled=0
+
+{"data": null, "errors": [{"message": "Syntax Error: Expected '{', found <EOF>.", "locations": [{"line": 1, "column": 7}]}]}%
+```
+
+It is recommended to understand the [common GraphQL error handling techniques](https://the-guild.dev/blog/graphql-error-handling-with-fp).
+
+# Up to 1000 for your query results
+As your entity lists grow, eventually they will be large enough such that rendering all of them will not be ideal. To optimize resource usage, Vendia now allows up to 1000 results being returned in a single query.
+
+Here are couple samples in a testing environment.
+
+1. A basic query with `limit` set to `1000`:
+
+```
+query MyQuery {
+  list_ProductItems(limit: 1000) {
+    nextToken
+    _ProductItems {
+      name
+    }
+  }
+}
+```
+
+You will see all or up to 1000 results which ever is lower:
+```
+{
+  "data": {
+    "list_ProductItems": {
+      "nextToken": null,
+      "_ProductItems": [
+        {
+          "name": "Item1"
+        },
+        {
+          "name": "item2"
+        }
+      ]
+    }
+  }
+}
+```
+
+2. A query attempt to set `limit` for more than `1000`:
+```
+query MyQuery {
+  list_ProductItems(limit: 1001) {
+    nextToken
+    _ProductItems {
+      name
+    }
+  }
+}
+```
+The query will then return null. It will provide you with the error message informing you that `the supplied list limit must be <= 1000`.
+```
+{
+  "data": {
+    "list_ProductItems": null
+  },
+  "errors": [
+    {
+      "message": "the supplied list limit must be <= 1000",
+      "locations": [
+        {
+          "line": 2,
+          "column": 3
+        }
+      ],
+      "path": [
+        "list_ProductItems"
+      ],
+      "extensions": {
+        "code": "VALIDATION_ERROR"
+      }
+    }
+  ]
+}
+```
+# Believe in Real Time Data Sharing?
```

Comment:
I *think* we use a hyphen for **`Real-Time`** in all instances but it's worth double-checking me on this.

Reply:
done